### ISSUE TITLE
Sorry guys, but...

### ISSUE THREAD

**Issue by remotevase** [Inactive-to-Reactive/ImagicalMine#33]
Sorry guys. As you may or may not have noticed, I have not been as intensive with my coding as I used to be. I have just been busier and busier. As the year goes on, I will have to become less and less active. I have lots going on, and I feel like I have taken on too much for me to actually follow through with. I will continue to test locally on my computer for fixes for 0.15, but will likely not be successful. If i am, i will certainly upload it to this repository. I will likely not have much time to work on many other projects. I hope that somebody will be there to continue some of my work. I have given my closest github friends (@taki21, @thedragonring, @imagicalgamer, and @applqpak) admin permissions on all of my repositories, so they can invite anybody. I have not had the time to add my new close friend on github (@ad5001) to my repositories, though... If someone asks for my help, please give them a link to this page. Also, **IF YOU WANT MY HELP, EMAIL ME AT remotevase@gmail.com!** Thank you! 
To summarize it all if you didnt want to read the whole thing, here are the bullet points:

- My closest friends have admin access to my repos
- I will not completely stop coding, and will only lessen the amount of time that I spend on it
- I am much busier
- If someone wants my help, they should email me at the email above
- Please help me by redirecting them to this page if they mention me
- I will frequently check my github notifications (3-4 times per week)
- HELP IS WANTED FOR PEOPLE TO ASSIST ME ON MY PROJECTS WHILE I AM TAKING MY BREAKS

Hope to see you all soon! This has been a wonderful experience!

**Comment by Sandertv** [comment]
😢

**Comment by Mike4l6** [comment]
ok

**Comment by Ad5001** [comment]
So sad D: ...
Since ImagicalMine was preaty much dead, we were almost the only ones to commit...
Now that ( :cry: ) how can IM survive? Exept if one of the team member fix the problem, I think it would be one of the hardest thing in the world to make IM survive.

Sad to see you go friend...

**Comment by HimbeersaftLP** [comment]
Nooooooooooooooo 😢

**Comment by matcracker** [comment]
I'm sorry, but when I saw how much the code are wrote bad, I said me by me "too much work re-write all, too many people that don't have knowlegement and put the hands in the source, so creating big mistakes". From those moment I didn't try to code.

My suggestion is: take the code of PocketMine (work better) and add ONLY the blocks/items, redstone of ImagicalMine. Then, we should create a new branch and try to fix all the issues and before merge it, check all the correct function.

**Comment by matcracker** [comment]
And... I forgot to say another thing. I want remeber that we should do a type of "Minecraft Server" and how I could see, you are adding strange commands or function. The point is, the plugin must do that, not the server (people can like it, but others, like me don't approve)!

**Comment by remotevase** [comment]
@matcracker thanks for the suggestion, but I do not think that this is the right place to put it. Anyways, I think that is a great idea. Maybe in the limited time that I spend on gh, I will follow through with that in my own repo. people who want to help can submit PR's

**Comment by TheProest35YT** [comment]
Remotevase since your server is in 0.15.0 but why no 15 phar

**Comment by remotevase** [comment]
Dude. Read the readme file. There is a Jenkins server that has the phar files.

**Comment by Ad5001** [comment]
@matcracker The thing you say is true. But if the commands are here, it's because they are essential or from MCPC vanilla or Buckit/Spigot.

**Comment by TheProest35YT** [comment]
I found out

**Comment by TheProest35YT** [comment]
Remote_vase why is your server offline

**Comment by remotevase** [comment]
@TheProest35YT major, MAJOR maintenance updates is why its offline

**Comment by remotevase** [comment]
Will be online in 2 weeks

**Comment by TheDragonRing** [comment]
😥 at least you're just becoming less active instead of completely leaving...

**Comment by TheProest35YT** [comment]
Where is IM Android i only find Genisys Android or PocketMine

**Comment by TheProest35YT** [comment]
And @remotevase your server has no plugins on minecraftpocket-servers.com

**Comment by remotevase** [comment]
> And @remotevase your server has no plugins on minecraftpocket-servers.com

1. STOP TALKING ABOUT MY SERVER! THIS IS NOT A CHAT FOR MY SERVER!!!!
2. REASON: minecraftpocket-servers.com uses query to recieve info about the server including players and plugins. When the server stops, the query is therefore cut off, and minecraftpocket-servers.com states that it is offline. If the server is stopped (is offline), and query is cut off, minecraftpocket-servers.com has no confirmation that there are plugins, so therefore, no plugins.

**Comment by TheDragonRing** [comment]
Why can't everyone just know how to use simple logic like that??? 😑

**Comment by ImagicalGamer** [comment]
:cry:

**Comment by da123rrell** [comment]
Why dont we add new younger coders to help replace some of the older and retired coders?

**Comment by TheProest35YT** [comment]
sorry @remotevase i didn't know LOL

**Comment by TheProest35YT** [comment]
anyone wanna join my group ? 

IMPlugins

**Comment by TheProest35YT** [comment]
@da123rrell im young but i don't know how to code i need to learn lol i learn from YouTube

**Comment by remotevase** [comment]
> Why dont we add new younger coders to help replace some of the older and retired coders?

I am still a young coder (Started a few months ago) yet I just dont have too much time...

> anyone wanna join my group ?
IMPlugins

Maybe, I'll take a look, but don't expect me to accept right away

**Comment by remotevase** [comment]
> anyone wanna join my group ?

> IMPlugins

Umm... It turns out that you are lying when you say 
> Only ImagicalMine plugin makers or chosen staff
Because you are **not** staff...

**Comment by daniktheboss** [comment]
Hey I can help! I am also a young coder! If you want to check my site! Fileduck.ddns.net! I would love to help and learn more with you!

**Comment by TheProest35YT** [comment]
@remotevase pls no copyrights for @IMPlugins its just a unofficial group 👍 
If u copyright @IMPlugins pls tell me

**Comment by TheProest35YT** [comment]
Report to me At either theproestyt35@gmail.com or imagicalplugins@gmail.com

**Comment by daniktheboss** [comment]
Wait this is now dead dead???

**Comment by Mike4l6** [comment]
no ones reposonded :/

**Comment by Ad5001** [comment]
It's not because we did not reply that we did not see it.

**Comment by TheProest35YT** [comment]
ImagicalMine still on Protocol 81

Latest Protocol : 82
ITXTech / Genisys is already done with 0.15.4

**Comment by remotevase** [comment]
Maybe that's one of the problem?

**Comment by TheProest35YT** [comment]
@IMPlugins Is Back Invitations Are Still Pending

**Comment by TheProest35YT** [comment]
Maybe
